### PR TITLE
[CopyFilesV2] Added retry count input.

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: CopyFilesV2

**Description**: Added retry count input.

**Documentation changes required:** (Y/N) Y

**Added unit tests:** (Y/N) N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/CopyFilesV2/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -19,6 +19,8 @@
   "loc.input.help.flattenFolders": "Flatten the folder structure and copy all files into the specified target folder.",
   "loc.input.label.preserveTimestamp": "Preserve Target Timestamp",
   "loc.input.help.preserveTimestamp": "Using the original source file, preserve the target file timestamp.",
+  "loc.input.label.retryCount": "Retry count to copy the file",
+  "loc.input.help.retryCount": "Specify the retry count to copy the file.",
```
Comment (anatolybolshakov) [inline Tasks/CopyFilesV2/Strings/resources.resjson/en-US/resources.resjson:23]: Could you please describe that this is more actual for unc network paths?

File: Tasks/CopyFilesV2/copyfiles.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ let targetFolder: string = tl.getPathInput('TargetFolder', true);
 let cleanTargetFolder: boolean = tl.getBoolInput('CleanTargetFolder', false);
 let overWrite: boolean = tl.getBoolInput('OverWrite', false);
 let flattenFolders: boolean = tl.getBoolInput('flattenFolders', false);
+let retryCount: number = parseInt(tl.getInput('retryCount'));
```
Comment (anatolybolshakov) [inline Tasks/CopyFilesV2/copyfiles.ts:22]: Could you please add default value here?

**Review by DaniilShmelev — APPROVED** [review]

**Review by EzzhevNikita — COMMENTED** [review]
Just to clarify, will we apply changes that related to Node 10 migration in separate PR?

**Comment by anatolybolshakov** [comment]
> Just to clarify, will we apply changes that related to Node 10 migration in separate PR?

Correct, we are planning to apply them separately

**Review by EzzhevNikita — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]

**Pull request merged**